dashbuilder/dashbuilder-client/dashbuilder-cms-client/src/main/java/org/dashbuilder/client/cms/screen/transfer/DataTransferExportPopUpView.html

```diff
@@ -0,0 +1,101 @@
+					<span class="spinner spinner-lg spinner-inline"></span> Loading
+					assets...
+				</p>
+			</div>
+			<div class="panel-group" id="dataSelectionAccordion">
+				<div class="panel panel-default">
+					<div class="panel-heading">
+						<h4 class="panel-title">
+							<a data-toggle="collapse" data-parent="#dataSelectionAccordion"
+								href="#collapseOne">DataSets</a>
+						</h4>
+					</div>
+					<div id="collapseOne" class="panel-collapse collapse in">
+						<div class="panel-body assets-panel-body">
+							<form role="form" class="search-pf has-button">
+								<div class="form-group has-clear">
+									<div class="search-pf-input-group">
+										<label for="searchDatasets" class="sr-only">Filter</label> <input
+											id="searchDatasets" type="search" class="form-control"
+											placeholder="Filter">
+
+									</div>
+								</div>
+							</form>
+							<!-- DataSets Table -->
+							<table class="table table-striped table-bordered table-hover"
+								id="datasetsTable">
+								<thead>
+									<tr>
+										<th><input type="checkbox" id="selectAllDatasets" /></th>
+										<th>UUID</th>
+										<th>Name</th>
+										<th>Type</th>
+									</tr>
+								</thead>
+								<tbody id="">
+								</tbody>
+							</table>
+						</div>
+					</div>
+				</div>
+				<div class="panel panel-default">
+					<div class="panel-heading">
+						<h4 class="panel-title">
+							<a data-toggle="collapse" data-parent="#dataSelectionAccordion"
+								href="#collapseTwo" class="collapsed">Pages</a>
+						</h4>
+					</div>
+					<div id="collapseTwo" class="panel-collapse collapse">
+						<div class="panel-body assets-panel-body">
+							<form role="form" class="search-pf has-button">
+								<div class="form-group has-clear">
+									<div class="search-pf-input-group">
+										<label for="searchPages" class="sr-only">Filter</label> <input
+											id="searchPages" type="search" class="form-control"
+											placeholder="Filter">
+
+									</div>
+								</div>
+							</form>
+							<!-- Pages Table -->
+							<table class="table table-striped table-bordered table-hover"
+								id="pagesTable">
+								<thead>
+									<tr>
+										<th><input type="checkbox" id="selectAllPages" /></th>
+										<th>Name</th>
+									</tr>
+								</thead>
+								<tbody></tbody>
+							</table>
+						</div>
+					</div>
+				</div>
+				<div class="panel panel-default">
+					<div class="panel-heading">
+						<h4 class="panel-title">
+							<a data-toggle="collapse" data-parent="#dataSelectionAccordion"
+								href="#collapseThree" class="collapsed">Navigation</a>
+						</h4>
+					</div>
+					<div id="collapseThree" class="panel-collapse collapse">
+						<div class="panel-body assets-panel-body">
+							<div class="input-group">
+								<div id="toggleSwitchContainer" class="form-control"
+									aria-describedby="navigationExportAddOn" style="height: auto"></div>
```

Comment:
The Toggle was removed.

We decided to remove the navigation selection because users should not opt the navigation out, otherwise components that refers to the navigation may break.